tools/clang/lib/CodeGen/CGHLSLMS.cpp

```diff
@@ -739,7 +739,7 @@ MDNode *CGMSHLSLRuntime::GetOrAddResTypeMD(QualType resTy) {
   case DXIL::ResourceClass::UAV: {
```

Comment:
This is part of `CGMSHLSLRuntime::GetOrAddResTypeMD` which I don't fully understand. It gets called when member functions of resource types get visited, but in that case we only have the `RecordType` available and not a variable declaration that might include `snorm`/`unorm`. It wasn't causing any errors in my testing though.